azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -193,7 +197,7 @@ public FlowRunnerManager(Props props, ExecutorLoader executorLoader,
         new JobTypeManager(props.getString(
             AzkabanExecutorServer.JOBTYPE_PLUGIN_DIR,
             JobTypeManager.DEFAULT_JOBTYPEPLUGINDIR), globalProps,
-            parentClassLoader);
+            getClass().getClassLoader());
```

Comment:
Don't understand. paranetClassLoader sounds indicating some class in upper level, not this clas?